source/api/events/window-event.md

```diff
@@ -0,0 +1,61 @@
+---
+title: window:&lt;event&gt;
+---
+
```

Comment:
What if their page has several `window`s, like if they have iframes on the page, can they listen to the iframe's window events? Can we show an example of this if so?

Reply:
No, it's the status quo as far as iframe support is concerned. It only binds to the AUT's top window. We'll have to do extra work when we add iframe support, but it shouldn't change the API. The window is available as `event.target`, so the user can know which iframe window it is based on that.